### PR TITLE
Login

### PR DESCRIPTION
**Pull request by marwajomaa**

### CODE REVIEW

File: client/src/Components/CommonComponents/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+class LogIn extends Component {
+  state={
+    loginInfo:{}
+  }
+
+  handleChange = (e)=>{
+    const { loginInfo } = this.state;
+    loginInfo[e.target.name]=e.target.value;
+    this.setState({loginInfo})
```
Comment (sultanassi95) [inline client/src/Components/CommonComponents/Login/index.js]:
```javascript
this.setState({
...this.state,
loginInfo
});
```

read about [it](https://medium.com/front-end-hacking/spread-operator-and-destructuring-arrays-and-objects-in-javascript-2f5578e1252b)

File: client/src/Components/CommonComponents/Footer/Style.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 * {
```
Comment (sultanassi95) [inline client/src/Components/CommonComponents/Footer/Style.css]: fix file name to `style.css`

File: client/src/Components/Pages/HomePage/HeroSection/index.js [hunk 3]
```diff
@@ -1,4 +1,3 @@
-
 import React from 'react';
 import './Style.css';
```
Comment (sultanassi95) [inline client/src/Components/Pages/HomePage/HeroSection/index.js]: don't forget to edit that after editing the file name as requested above.

File: client/src/Components/CommonComponents/Login/index.js (hunk 1 above)
Comment (marwajomaa) [reply client/src/Components/CommonComponents/Login/index.js]: is there any problem with this code need to be fixed??

File: client/src/Components/CommonComponents/Footer/Style.css (hunk 2 above)
Comment (marwajomaa) [reply client/src/Components/CommonComponents/Footer/Style.css]: I didn't work on this section so I don't know if I have the right to modify it, can I ?? @hshahwan

Comment (hshahwan) [reply client/src/Components/CommonComponents/Footer/Style.css]: Hi @marwajomaa I can change it, don't worry about that :)

Comment (hshahwan) [reply client/src/Components/CommonComponents/Footer/Style.css]: Its ok you change it

File: client/src/Components/CommonComponents/Login/index.js (hunk 1 above)
Comment (sultanassi95) [reply client/src/Components/CommonComponents/Login/index.js]: replace the code you are using `this.setState()` 
with the code I suggested.
and read about the reason why should you do that in that [link](https://medium.com/@thejasonfile/using-the-spread-operator-in-react-setstate-c8a14fc51be1)

Comment (sultanassi95) [reply client/src/Components/CommonComponents/Login/index.js]: @marwajomaa

File: client/src/Components/App.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import ResultPage from './Pages/ResultPage';
 import Signup from './Pages/Signup';
 import Header from './CommonComponents/Header';
 import './App.css';
+import Login from './CommonComponents/Login'
 
 class App extends Component {
   state = {
```
Comment (RamyAlshurafa) [inline client/src/Components/App.js]: Where is the state ??!!

File: client/src/Components/CommonComponents/Footer/Style.css (hunk 2 above)
Comment (sultanassi95) [reply client/src/Components/CommonComponents/Footer/Style.css]: @marwajomaa please change that file name :cry:

Comment (marwajomaa) [reply client/src/Components/CommonComponents/Footer/Style.css]: I change it but it's not appear

Comment (sultanassi95) [reply client/src/Components/CommonComponents/Footer/Style.css]: @marwajomaa did u make sure that it's correctly 1. added 2. committed and 3. pushed.

**Pull request merged**